### ISSUE TITLE
Associate GitHub account with new username

### ISSUE THREAD

**Issue by hanggrian** [gradle/plugin-portal-requests#227]
Hi Gradle Plugin support,

I have created a new username __hanggrian__ using the email address __hanggrian@proton.me__. However, my GitHub account is still linked to the old username __hendraanggrian__. How can I transfer the GitHub account to my new username?

### Plugin Id

I don't wish to transfer ownership of the plugins because I have changed the namespace of my plugin IDs. I will request to remove them once I publish the plugins with my new username.

### Current Plugin Owner

hendraanggrian

### Expected Plugin Owner

hanggrian

### Context

I failed to update my domain registration which resulted in my URL being taken. I have registered a new domain and changed my GitHub username to reflect my new domain. Seeing that I am unable to change my Gradle username, I have also created a new account using __hanggrian__ username.

**Comment by ov7a** [comment]
Hi!

We need to verify your identity. Can you confirm that change by [contacting us](mailto:plugin-portal-support@gradle.com) using your **old** e-mail?

Once that is done, we can change the Github association. However, we won't delete the plugins - they have a number of downloads, and it's against our policy to delete plugins because it may break builds.

You can publish them under a new group, and we can update the description for the old plugins to point to the new ones.

**Comment by hanggrian** [comment]
Hello there, thanks for your insight on Gradle plugin removal! When the migration is complete, I will open a new issue to update the descriptions of my old plugins.

I have also sent the email using my old email address.

Thanks again for your help.

**Comment by ov7a** [comment]
The Github association is completed.
